### ISSUE TITLE
New Feature: Display template preview in template editor

### ISSUE THREAD

**Issue by mattvanniekerk** [jeffream/csci-401-capstone#9]
<img id="footer-preview" src="#" alt="">

**Comment by AlexVolcy** [comment]
Priority: 3, Severity 4

**Comment by jtejada123** [comment]
Is this the same as issue 27? If so my apologies I'll go ahead and close the other one

**Comment by mattvanniekerk** [comment]
Not exactly. #27 is about uploading files; this is about displaying a preview of the uploaded file. #27 needs to work before this one can. However, the plan now is to upload only one file, a template.docx. Do we still want to display a file preview?

**Comment by jtejada123** [comment]
Gotcha. I think it doesn't hurt to be able to preview the uploaded file, but if it ends up being too complicated we'll scrap it because its not all that necessary.

**Comment by mattvanniekerk** [comment]
This bug is not exactly applicable to the new method of template creation—uploading a .docx file. It may be something we would like to implement as a new feature though.
